packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -105,6 +187,22 @@ class HorizontalOverflow extends Foundation<HorizontalOverflowProps,  React.AllH
         }
     }
 
+    private renderPreviousButton(): JSX.Element {
+        return (
+            <div className={this.props.managedClasses.horizontalOverflow_previous} onClick={this.handlePreviousClick}>
+                {this.getChildBySlot(ButtonDirection.previous)}
+            </div>
+        );
+    }
+
+    private renderNextButton(): JSX.Element {
+        return (
+            <div className={this.props.managedClasses.horizontalOverflow_next} onClick={this.handleNextClick}>
```

Comment:
I think I would prefer to see these buttons not abstracted to a function - the function doesn't perform any logic and reading the render function now makes me track down what exactly `renderNextButton` does. I think it's more simple to just keep this in the render function.

Reply:
This was actually part of another task @janechu for guidance